README.md

```diff
@@ -29,6 +30,19 @@ Table of Contents
 * [Running tests](#running-tests)
 * [Contributing](#contributing)
 
+## Intended Audience
+
+- Running the Cloudera Distribution of Hadoop
+- Orchestrating the cluster with Cloudera Manager
+  - Comfortable with CSD's
+- Support applications on the cluster
+- Desire aggregate logging solution for applications
+- Turn application logs into actionable data points
+  - Build application dashboards
+  - Enable custom log search
+  - Alert when critical criteria is met
+- Comfortable applying Log4j components
+- 
```

Comment:
@KeithSSmith what do you think about something like this, I took what you had and made it into a paragraph:
```
Pulse is a Hadoop native framework that enhances application support.  Pulse will aggregate application logs for easy searching and add pro-active alerting to your applications. A Cloudera Custom Service Descriptor (CSD) is included so Cloudera Manager can be used for installation, management, and monitoring of the Pulse components. 
```